lxd/db/images.go

```diff
@@ -556,6 +558,63 @@ func (c *Cluster) GetImageFromAnyProject(fingerprint string) (int, *api.Image, e
 	return object.ID, &image, nil
 }
 
+func (c *ClusterTx) getImagesByShortformFingerprint(fingerprint string, project *string, public *bool) ([]Image, error) {
```

Comment:
@masnax please can we rename this function to `getImagesByFingerprintPrefix` and then the `fingerprint` argument can become `fingerprintPrefix`. Then the wildcard can be added to the passed in string inside this function rather than in `GetImage`. This way it keeps the prefix query logic inside this function, as well as self describing the so-called "short form" fingerprint format.